api/src/data_migration/mock/generate.py

```diff
@@ -0,0 +1,112 @@
+#
+# Populate source tables with mock data.
+#
+
+import logging
+
+import factory
+import sqlalchemy
+
+import src.adapters.db as db
+import src.adapters.db.flask_db as flask_db
+from src.data_migration.data_migration_blueprint import data_migration_blueprint
+from tests.src.db.models.factories import ForeignTopportunityFactory
```

Comment:
This might hit some issues non-locally if merged. The factory libraries are dev dependencies, and I think that would cause issues in the non-local builds.

Reply:
To work around that for the seed script I built, I just don't use flask and manually setup the logging/db session: https://github.com/HHS/simpler-grants-gov/blob/main/api/tests/lib/seed_local_db.py

Only adds a few lines.